packages/@ionic/cli/src/lib/integrations/capacitor/index.ts

```diff
@@ -190,7 +191,18 @@ export class Integration extends BaseIntegration<ProjectIntegration> {
 
     debug('Getting config with Capacitor CLI: %O', args);
 
-    const output = await this.e.shell.cmdinfo('capacitor', args, { cwd: this.root });
+    const output = await (async (_command: string, _args: readonly string[] = [], opts: IShellSpawnOptions = {}) => {
+      const proc = await this.e.shell.createSubprocess(_command, _args, opts);
+
+      try {
+        const out = await proc.output();
+        return out.split('\n').join(' ').trim();
+      } catch (e) {
+        return '';
```

Comment:
shouldn't it return something?
In example if I uninstall @capacitor/cli when it runs `capacitor sync android` I just get
```
Error

at new SubprocessError
...
(stack trace of files and lines)
```